lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/aplinear/APLExplanation.java

```diff
@@ -32,7 +37,13 @@ public APLExplanation(
         return results;
     }
 
-    String prettyPrint() {
+    @JsonProperty("numTotal")
```

Comment:
json serialization isn't implemented on the actual APLExplanationResult yet so when you run the server and cal

curl -H "Accept: application/json" -H "Content-type: application/json" \
-X POST -d "@core/demo/cli_meanshift_rest.json" 0.0.0.0:4567/query; echo

there is an error. Need to implement the json serialization in the new aplexplanation class. Sorry I know it's annoying and thanks a lot!